solr/solrj/ivy.xml

```diff
@@ -39,6 +39,7 @@
     <dependency org="org.apache.httpcomponents" name="httpcore" rev="${/org.apache.httpcomponents/httpcore}" conf="compile"/>
     <dependency org="commons-io" name="commons-io" rev="${/commons-io/commons-io}" conf="compile"/>
     <dependency org="org.apache.commons" name="commons-math3" rev="${/org.apache.commons/commons-math3}" conf="compile"/>
+    <dependency org="org.apache.commons" name="commons-compress" rev="${/org.apache.commons/commons-compress}" conf="compile"/>
```

Comment:
usually, we face a lot of resistance to adding dependencies to SolrJ. Is it possible to use any compression that is natively available to java?

Reply:
Yes, I think so, testing out with native compression now.

Reply:
Modified this to use native Java compression